app/src/components/popups/StopPopup.vue

```diff
@@ -152,49 +153,37 @@ const augmentedArrivals = computed<(Omit<StopArrival, 'eta'> & { nextStopName?:
       return null;
     }
 
-    return stop.value.arrivals
-      .toSorted((a, b) => {
-        if (a.eta === 0 || b.eta === 0) {
-          return a.planned.localeCompare(b.planned);
-        }
-        return a.eta - b.eta;
-      })
-      .map((a) => {
-        const trip = trips.value[a.tripId];
-
-        let nextStopName: string | undefined;
-        if (trip !== undefined && trip.arrivals !== undefined) {
-          const nextStopIndex = trip.arrivals.findIndex((s) => s.id === props.marker.id);
-          if (nextStopIndex !== -1) {
-            nextStopName = trip.arrivals[nextStopIndex + 1]?.name;
+    return (
+      stop.value.arrivals
+        .toSorted((a, b) => {
+          if (a.eta === 0 || b.eta === 0) {
+            return a.planned.localeCompare(b.planned);
           }
-        }
-
-        return {
-          ...a,
-          nextStopName,
-          eta: eta(a),
-        };
-      });
+          return a.eta - b.eta;
+        })
+        // eslint-disable-next-line arrow-body-style
+        .map((a) => {
+          // const trip = trips.value[a.tripId];
+
+          // let nextStopName: string | undefined;
+          // if (trip !== undefined && trip.arrivals !== undefined) {
+          //   const nextStopIndex = trip.arrivals.findIndex((s) => s.id === props.marker.id);
+          //   if (nextStopIndex !== -1) {
+          //     nextStopName = trip.arrivals[nextStopIndex + 1]?.name;
+          //   }
+          // }
```

Comment:
@anbraten Does this feature not work with the new api?

Reply:
@lukashass The nextStop feature was disabled in general, so I commented this code piece for now as well.

Reply:
I think it was still active, just relying on the data already loaded by viewing a bus before viewing a stop. But it's not too important.